bitnami/etcd/templates/statefulset.yaml

```diff
@@ -143,13 +135,7 @@ spec:
             - name: ETCD_INITIAL_CLUSTER_TOKEN
               value: "etcd-cluster-k8s"
             - name: ETCD_INITIAL_CLUSTER_STATE
-            {{- if not (empty .Values.etcd.initialClusterState) }}
-              value: {{ .Values.etcd.initialClusterState | quote }}
-            {{- else if .Release.IsInstall }}
-              value: "new"
-            {{- else }}
-              value: "existing"
-            {{- end }}
+              value: {{ ternary .Values.initialClusterState (ternary "new" "existing" .Release.IsInstall) (not (empty .Values.initialClusterState)) | quote }}
```

Comment:
is necessary checking the not empty in initialClusterState? wouldn't just doing .Values.initialClusterState suffice?

Reply:
I'll use `default`